catroid/src/main/res/xml/preferences.xml

```diff
@@ -64,12 +64,13 @@
 
     </PreferenceScreen>
 
-    <PreferenceScreen
-            android:key="setting_parrot_jumping_sumo_bricks"
-            android:summary="@string/preference_description_jumpingsumo_bricks"
-            android:title="@string/preference_title_enable_jumpingsumo_bricks" >
+    <CheckBoxPreference
+        android:defaultValue="@string/FEATURE_ARDUINO_PREFERENCES_ENABLED"
```

Comment:
The jumping sumo feature should not be enabled by the arduino flavor itself. so let it per default false.